### DISCUSSION TITLE
Can't Create a Custom Scope with Many Repos

### DISCUSSION THREAD

**Discussion by timothysteward-wk** [community/community#12255] [Other Feature Feedback, Questions, & Ideas] [answered]
If I try to create a custom scope in Github Code Search, and I add too many items, the save button is pushed off screen and not accessible.

I have a work around: I go to the advanced tab, and click save, but then it a seems it truncates to the first 15 repos.

I am on a team that works in a large repository that consumes many dependencies internal to my organization. I want to be able to limit my search to this repository and it's direct dependencies, but I can't because of this limitation.

**Comment by Avivhdr** [comment]
Hi Tim, See this ticket I opened a while ago about it.
https://github.com/github/feedback/discussions/11714

Git it a ⬆️  so it would get noticed.

**Comment by colinwm** [comment, marked as answer]
Hey @timothysteward-wk, thanks for the report. I wrote a more detailed response in https://github.com/github/feedback/discussions/11714, but the UI issue you describe should now be fixed.
